inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js

```diff
@@ -462,7 +462,7 @@
     events = [];
     for (sel in eventsObj) {
       functionName = eventsObj[sel];
-      _ref1 = sel.split(' '), selector = 2 <= _ref1.length ? __slice.call(_ref1, 0, _k = _ref1.length - 1) : (_k = 0, []), event = _ref1[_k++];
```

Comment:
This is a generated file (see comments at the beginning of file), hence refactoring should be done in the code that generates this file. However, since this is 3rd party code, I don't think this is possible.

Reply:
There should be no changes in this file.

Reply:
Changes in this file are undone